docs.feldera.com/docs/get-started/enterprise/authentication/okta-sso.md

```diff
@@ -0,0 +1,297 @@
+   - **Include in token type**: `Access Token`
+   - **Value type**: `Groups`
+   - **Value**: Select appropriate group filter or use regex `.*`
+   - **Include in**: `Any scope`
+
+## Tenant Assignment with a custom claim
+
+Feldera supports [multiple tenant assignment strategies](index.mdx#Tenant Assignment Strategies).
+
+### Individual Tenancy (Development)
+
+No additional Okta configuration required.
+
+### Organization Tenancy (One tenant per org)
+
+No additional Okta configuration required.
+
+### Custom Tenant Claims (Enterprise)
+
+Use the custom authorization server configured above to assign users to specific tenants based on their group membership.
+
+Below is an example configuration that uses the tenant claim based on the user's Feldera-specific group membership. Users should be assigned to groups with a `feldera_` prefix to distinguish them from other organizational groups. For example:
+- User in group `feldera_engineering` → `tenant` claim = `feldera_engineering`
+- User in group `feldera_marketing` → `tenant` claim = `feldera_marketing`
+- User in group `feldera_customer_acme` → `tenant` claim = `feldera_customer_acme`
+
+#### Step 1: Create Tenant Groups
+
+In Okta **Directory** → **Groups**:
+
+1. Create groups using the `feldera_` prefix followed by the tenant name:
+2. Examples:
+   - `feldera_engineering`
+   - `feldera_marketing`
+   - `feldera_customer_acme`
+   - `feldera_customer_globex`
+
+#### Step 2: Assign Users to Groups
+
+Add users to appropriate Feldera tenant groups based on their access requirements. Users can belong to multiple groups, but only groups with the `feldera_` prefix will be considered for tenant assignment.
+
+#### Step 3: Configuring Feldera instance
+
+The `tenant` claim is always respected, so you only need to disable individual tenancy:
+
+**Feldera Configuration:**
+```bash
+pipeline-manager ...\
+  --auth-provider=okta \
+  --individual-tenant=false
+```
+
+## Group membership authorization with a custom claim
+
+Feldera can restrict access based on Okta group membership using the `groups` claim configured in your custom authorization server. This is separate from tenant assignment and controls who can access Feldera at all.
+
+The groups claim is automatically populated in the custom authorization server setup above. You can adjust the group filter in the claim configuration if needed.
+
+### Configure Feldera Authorization
+
+```bash
+# Require users to belong to specific groups
+pipeline-manager \
+  --auth-provider=okta \
+  --authorized-groups=feldera-users,analytics-team
+```
+
+Users must belong to at least one of the specified groups to access Feldera. If `--authorized-groups` is not specified, no group restrictions apply.
+
+## Environment Variables
+
+Configure the following environment variables for your Feldera deployment:
+
+### Required Variables
+
+```bash
+# Okta OIDC configuration
+AUTH_ISSUER=https://<your-okta-domain>/oauth2/<custom-auth-server-id>
```

Comment:
these we dont pass as config variables? maybe they should appear in config too and ten env variables prefixed with FELDERA_AUTH_*